### DISCUSSION TITLE
Consulta SQL no PHP

### DISCUSSION THREAD

**Discussion by jcvilanova** [frontendbr/forum#2287] [Perguntas e Respostas] [answered]
Galeara, o que há de errado na instrução abaixo: 
```sql
SELECT
  art.id AS id_art,
  art.titulo AS tit_art,
  art.descricao AS desc_art,
  art.conteudo AS cont_art,
  art.imagem AS img_art,
  art.slug AS slg_art, 
  sts_cats_artigo_id AS art.cat_id,
  cats.id AS id_cat,
  cats.nome AS nome_cat
FROM sts_artigos AS art
INNER JOIN sts_cats_artigos AS cats ON cats.id=art.cat_id 
WHERE sts_cats_artigo_id=1 ORDER BY id ASC
LIMIT 8
```
 
Estou querendo resgatar todos os artigos da categoria 1, na tabela sts_artigos temos o campo sts_cats_artigo_id e na tabela sts_cats_artigos temos o campo ID. Dessa forma como quero recuperar o nome da categoria, penso ser necessário usar o INNER JOIN, porem, com a clausula WHERE. Quando coloquei apenas o INNER JOIN resgatou porem, ao colocar o WHERE o array veio vazio. 
O que pode estar errado ?

**Comment by felquis** [comment, marked as answer]
Eai bele? Eu arranho no SQL. 

Mas olhando sua query parece que a linha `sts_cats_artigo_id AS art.cat_id` está invertida, e deveria ser `art.cat_id AS sts_cats_artigo_id`.

Não sei se isso resolve, mas é algo a ser corrigido sem dúvidas.

**Reply by jcvilanova** [reply]
Corrigiu, obrigado
